projects/cardiovascular-patient-app/src/app/app.component.ts

```diff
@@ -21,4 +22,8 @@ export class AppComponent implements OnInit {
       layers.forEach((layer) => this.layerManager.select(layer.id));
     });
   }
+
+  getSelectedIndex(index: any) {
+    this.selectedIndex = index
+  }
```

Comment:
please rename this setter to avoid confusion, or just set the value directly in the template